pkg/asset/installconfig/ovirt/validaton.go

```diff
@@ -125,3 +127,20 @@ func validateVNICProfile(platform ovirt.Platform, con *ovirtsdk.Connection) erro
 	}
 	return nil
 }
+
+func validURL(val interface{}) error {
```

Comment:
Question: validURL shouldn't be string  as it's an URL?

Reply:
This is the signature defined as Validator type by the survey project 

https://github.com/AlecAivazis/survey/blob/6773bdf39b7fa13e6a40ece7ac2d01e0d469c205/survey.go#L27

and allowed to be used in the composeValidators variadic function

https://github.com/AlecAivazis/survey/blob/6773bdf39b7fa13e6a40ece7ac2d01e0d469c205/validate.go#L62